docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -0,0 +1,70 @@
+---
+approvers:
+- mtaufen
+- dawnchen
+title: Set Kubelet parameters via a config file
+---
+
+{% capture overview %}
+{% include feature-state-alpha.md %}
+
+As of Kubernetes 1.8, a subset of the Kubelet's configuration parameters may be
+set via an on-disk config file, as a substitute for command-line flags. In the
+future, most of the existing command-line flags will be deprecated in favor of
+providing parameters via a config file, which simplifies node deployment.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+- A v1.8 or higher Kubelet binary must be installed.
+
+{% endcapture %}
+
+{% capture steps %}
+
+## Create the config file
+
+The subset of the Kubelet's configuration that can be configured via a file
+is defined by the `KubeletConfiguration` struct
+[here (v1alpha1)](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/kubeletconfig/v1alpha1/types.go).
+The configuration file must be a JSON or YAML representation of the parameters
+in this struct. Note that that this structure, and thus the config file API,
+is still considured alpha and is not subject to stability gurarantees.
+
+Create a file named `kubelet` in its own directory and make sure the directory
+and file are both readable by the Kubelet. You should write your intended
+Kubelet configuration in this `kubelet` file.
+
+For a trick to generate a configuration file from a live node, see
+[Reconfigure a Node's Kubelet in a Live Cluster](/docs/tasks/administer-cluster/reconfigure-kubelet).
+
+## Start a Kubelet process configured via the config file
+
+Start the Kubelet with the `KubeletConfigFile` feature gate enabled and the 
+Kubelet's `--init-config-dir` flag set to the location of the directory
+containing the `kubelet` file. The Kubelet will then load the parameters defined
+by `KubeletConfiguration` from the `kubelet` file, rather than from their 
+associated command-line flags.
```

Comment:
so settings in the configuration file take precedence of the command line flags?